kubernetes-cronjob/massdriver.yaml

```diff
@@ -5,71 +5,191 @@ source_url: github.com/YOUR_ORG/<md .Name md>
+          description: The expected Memory resources required for this application.
+          type: integer
+    job:
       type: object
+      title: Job Settings
+      required:
+        - schedule
+        - concurrencyPolicy
+        - completions
+        - parallelism
       properties:
-        enabled:
-          type: boolean
-          default: false
-        minReplicas:
+        schedule:
+          title: Schedule
+          description: The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+          type: string
+          default: "* * * * *"
+          pattern: ^(@(annually|yearly|monthly|weekly|daily|hourly))|((((\d+,)+\d+|((\d+|\*)(\/|-)\d+)|\d+|\*) ){4}((\d+,)+\d+|((\d+|\*)(\/|-)\d+)|\d+|\*))$
+          message:
+            pattern: "Value must be in the standard cron format: See https://en.wikipedia.org/wiki/Cron"
+        concurrencyPolicy:
+          title: Concurrency Policy
+          description: "Specifies how to treat concurrent executions of a Job. This occurs when the previous job hasn't completed before the next job is scheduled. Valid values are: - 'Allow': allows CronJobs to run concurrently; - 'Forbid': forbids concurrent runs, skipping next run if previous run hasn't finished yet; - 'Replace': cancels currently running job and replaces it with a new one"
+          type: string
+          default: Allow
+          enum:
+            - Allow
+            - Forbid
+            - Replace
+        completions:
+          title: Completions
+          description: Specifies the number of successfully finished pods the job requires in order to be considered complete.
           type: integer
           default: 1
-        maxReplicas:
+          minimum: 1
+        parallelism:
+          title: Parallelism
+          description: Specifies the maximum number of pods the job should run in parallel at any given time. Must be less than or equal to Completions.
           type: integer
-          default: 2
-        targetCPUUtilizationPercentage:
-          type: integer
-          default: 90
-    envs:
-      type: array
-      items:
-        type: object
-        properties:
-          name:
-            type: string
-          value:
-            type: string
-<md if not .Connections md>
-connections:
-  properties: {}
-<md end -md>
+          default: 1
+          minimum: 1
+    # Uncomment the section below to begin adding any application-specific configuration that is appropriate. See the app.envs section above for how to set environment values from these settings.
+    # configuration:
+    #   title: Configuration
+    #   description: Application specific configuration settings
+    #   type: object
+    #   properties:
+    #     log_level:
+    #       title: Log Level
+    #       type: string
+
 
-<md if .Connections md>
 connections:
   required:
+    - kubernetes_cluster
   <md- range $key, $art:= .Connections md>
     - <md $key md>
   <md- end md>
+  oneOf:
```

Comment:
This doesn’t work. Remove the oneOf.required. We just left them all optional in deployments

Reply:
See #28 for notes. It’s fine to leave them all optional for now until we find a fix to apply across the board.